osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/server/profile/Smr5Profile.java

```diff
@@ -214,4 +259,109 @@ public ConfigurationObject configurationObject() {
   public Long mbusIdentificationNumberHolder() {
     return this.mbusIdentificationNumber;
   }
+
+  @Bean
+  public DoubleLongUnsignedExtendedRegister mBusScalerUnitVolumeHourlyChannel1() {
```

Comment:
I think `ScalerUnit` is not needed in the name. `mBusVolumeHourlyChannel1` would be clearer.